app/views/pages/index.html.slim

```diff
@@ -1,16 +1,15 @@
 - unless user_signed_in?
-  h1 Help your favorite open source projects l
-  p Sign up to receive a different open issue in your inbox every day from your favorite GitHub repos
-  p #{number_with_delimiter(User.count, delimiter: ',')} developers are working on #{Repo.count} open source repos
-  = link_to 'Sign up today', new_user_registration_path
-
+  .hero
+    h1.hero--primary-title Help your favorite open source projects and become a better developer while doing it.
```

Comment:
`.hero-title--primary` might work better here. and `.hero-title--secondary` below